sympy/matrices/solvers.py

```diff
@@ -699,6 +708,67 @@ def _pinv_solve(M, B, arbitrary_matrix=None):
             A_pinv.multiply(A)).multiply(arbitrary_matrix)
 
 
+def _cramer_solve(M, rhs, det_method="bird"):
+    """Solves system of linear equations using Cramer's rule.
```

Comment:
You could cite the wikipedia article for Cramer's rule.

Reply:
Thanks, addressed in c62209b61136b17180a00a2de8a6553eae208461